models/aether-2.1.x/yang/onf-device.yang

```diff
@@ -54,6 +54,21 @@ submodule onf-device {
         description
           "Link to simcard";
       }
+
+      container state {
+        config false;
+        description "read only attributes of device";
+
+        leaf ip-address {
+          type string;
+          description "the ip address assigned to the device";
+        }
+
+        leaf connected {
+          type string;
```

Comment:
If it's only two states, would a boolean be better? Also, it might be tri-state "Yes | No | Idle". That could be a question for Ajay. Finally, the time of last connection might also be interesting. To include times would stumble into the territory of metrics, but I do wonder if it would be useful to have limited timestamps here, like "Last Connected".

Reply:
I will add `last-connected`.

Reply:
Re the `Yes | No | Idle` - in one way it's good to leave the defintion vague i.e. string because it can be changed at runtime - in another way it's a bad thing that it could change at runtime, and it does not help code generation.

I will ask Ajay